Sources/__SwiftOnServer_org/Documentation.docc/2024/02/08-using-swiftnio-channels/08-using-swiftnio-channels.md

```diff
@@ -1,23 +1,8 @@
----
-slug: using-swiftnio-channels
-title: Using SwiftNIO - Channels
-description: Create a TCP server using SwiftNIO and structured concurrency
-date: 2024/02/08
-tags: Swift, SwiftNIO, Networking
-author: Joannis Orlandos
-authorLink: https://x.com/JoannisOrlandos
-authorGithub: joannis
-authorAbout: Joannis is a seasoned member of the Swift Server WorkGroup, and the co-founder of Unbeatable Software B.V. If you're looking to elevate your team's capabilities or need expert guidance on Swift backend development, consider hiring him.
-cta: Get in touch with Joannis
-ctaLink: https://unbeatable.software/mentoring-and-training
-company: Unbeatable Software B.V.
-companyLink: https://unbeatable.software/
-duration: 30 minutes
----
-
 # SwiftNIO Channels
 
-In the [previous tutorial](/using-swiftnio-fundamentals), you've learned the fundamentals of SwiftNIO. You're now familiar with the concept of an `EventLoop`.
+SwiftNIO revolves around networking. In SwiftNIO, reads and writes happen using a ``Channel``.
+
+In the [previous tutorial](/using-swiftnio-fundamentals), you've learned the fundamentals of SwiftNIO. You're now familiar with the concept of an ``EventLoop``.
```

Comment:
this link is missing the numeric prefix `30-`

Reply:
No, the links on the site don't include the date. Maybe it's a good idea to align those.

Reply:
Hrm, links between articles are broken

Reply:
i'm not sure how those links resolved previously, do they work in DocC?

Reply:
I didn't try, so I assume not

Reply:
My goal was to link across the final rendered pages, not between docc files

Reply:
alternatively, you could rename the article files themselves so that they no longer contain the numeric prefixes. would that work here?

Reply:
i think the Right Thing to do here is to 1) rename the articles so that the filename matches the desired URL path component, which will allow you to use unidoc/DocC's validation and diagnostic features and 2) change the renderer default to emit relative URL references for articles within the same package.

if you are using links to section headings, this will also allow you to validate the section links, which you won't be able to do easily with absolute URLs.

would that work for you?

Reply:
Works perfectly fine! I'll update